lib/CUFFT_Plans.h

```diff
@@ -4,7 +4,7 @@
 
 #include <quda_internal.h>
 #include <quda_matrix.h>
-#include <cufft.h>
+#include <quda_fft.h>
```

Comment:
Is this the correct place to abstract, e.g., within the CUFFT_Plans.h file?  I could imagine that the CUFFT_Plans.h file would be unique to the CUDA target and we would have different plan headers for other arches.

Reply:
Gah! I forgot to omit this from the auto converter. I'm sure that ant other target will have it's own FFT implementation.